src/hooks/useEditVideoSheet.tsx

```diff
@@ -180,19 +203,27 @@ export const useEditVideoSheetTabData = (tab?: EditVideoSheetTab) => {
     activeUser: { channelId },
   } = useActiveUser()
   const { drafts } = useDrafts('video', channelId ?? '')
-
-  const { selectedVideoTabCachedAssets } = useEditVideoSheet()
+  const { selectedVideoTabCachedAssets, selectedVideoTabCachedDirtyFormData } = useEditVideoSheet()
 
   const { video, loading, error } = useVideo(tab?.id ?? '', { skip: tab?.isDraft })
 
   if (!tab) {
     return {
-      data: null,
+      updateTabData: () => ({}),
```

Comment:
I'm a little confused here, why we need this? Could you explain?

Reply:
oops, that was not meant to stay there, good catch 👌